src/pkgcheck/checks/perl.py

```diff
@@ -22,6 +25,24 @@ def desc(self):
         return f"DIST_VERSION={self.dist_version} normalizes to {self.normalized}"
 
 
+class MissingOPVirtualPerlDependency(results.VersionResult, results.Warning):
```

Comment:
Maybe:
```suggestion
class MissingVersionedVirtualPerlDependency(results.VersionResult, results.Warning):
```

although it's a bit long?